lint/ruff.bzl

```diff
@@ -90,7 +90,7 @@ def ruff_fix(ctx, executable, srcs, config, patch):
         output = patch_cfg,
         content = json.encode({
             "linter": executable._ruff.path,
-            "args": ["check", "--fix"] + [s.short_path for s in srcs],
+            "args": ["check", "--fix"] + [s.path for s in srcs],
```

Comment:
Why was this needed?

Reply:
Greg asked for it on #90 :)
It's unrelated.